genrules.cpp

```diff
@@ -2477,7 +2477,11 @@ int Game::GenRules(const AString &rules, const AString &css,
 		temp += ItemDefs[last].names;
 		temp += " for each ";
 		temp += Globals->UPKEEP_FOOD_VALUE;
-		temp += " silver of maintenance owed. ";
+		if (Globals->MAINTENANCE_COST % Globals->UPKEEP_FOOD_VALUE)
```

Comment:
we should probably check if UPKEEP_FOOD_VALUE is not 0 (in case some game will be no upkeep)

Reply:
If `UPKEEP_FOOD_VALUE` will be 0, engine will crash